pubspec.yaml

```diff
@@ -43,4 +43,7 @@ dev_dependencies:
 
 
 flutter:
-  uses-material-design: true
\ No newline at end of file
+  uses-material-design: true
+
+  assets:
+    - assets/images/vost_logo_white.png
```

Comment:
Se este logo não for usado dentro da app e for só no splash nativo, não é necessário estar aqui. Mas acredito que seja usado, é só para confirmar.

Reply:
I left it out since it really might be used inside the app, also, this way, it is the source of truth for the Android and iOS native splash.
Can we left it?

Reply:
Sure! That's what I thought.